packages/app/plugin/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ConfigPlugin, withPlugins, createRunOncePlugin } from '@expo/config-plugins';
+
+import {
+  withApplyGoogleServicesPlugin,
+  withBuildscriptDependency,
+  withCopyAndroidGoogleServices,
+} from './android';
+import { withFirebaseAppDelegate, withIosGoogleServicesFile } from './ios';
+
+const DEFAULT_ANDROID_GOOGLE_SERVICES_PATH = './google-services.json';
+const DEFAULT_IOS_GOOGLE_SERVICES_PATH = './GoogleService-Info.plist';
+
+interface PluginProps {
+  /**
+   * Custom location of `google-services.json`,
+   * relative to project root
+   */
+  androidGoogleServicesPath?: string;
```

Comment:
For now, maybe just rely on the Expo CLI built-in values since they could be used across Google Sign-in and other packages. This will reduce initial complexity in the config plugin as well.